src/pyOpenMS/tests/unittests/test_docstring_format.py

```
@@ -0,0 +1,1403 @@
+            docstrings = extract_docstrings_from_pyx(pyx_file)
+
+            for name, doc_type, docstring, line_num in docstrings:
+                if doc_type == 'class':
+                    # Check if docstring is essentially empty or just whitespace
+                    content = docstring.strip()
+                    if not content or content == '""':
+                        empty_docs.append(f"{filename}:{line_num} ({name})")
+
+        # Allow some empty docs (internal classes) but not too many
+        if len(empty_docs) > 50:
+            assert False, (
```

Ruff (0.14.11)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)